### PR TITLE
Creating Tab component

### PR DESCRIPTION
**Pull request by sallamTanna**
- This is the material-UI  "Tab" component. Just edited its style when the tab is active (#22)

### CODE REVIEW

File: client/src/components/Tab/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    '&$selected': {
+      background: '#ED2585',
+      color: 'white',
+      fontWeight: 'bold'
+    },
+  },
+  selected: {},
+});
+
+export default function Tab(props) {
```
Comment (AbdallahAmmar96) [inline client/src/components/Tab/index.js]: The function name is the same name in line 3

Comment (sallamTanna) [reply client/src/components/Tab/index.js]: Edited

**Comment by AbdallahAmmar96** [comment]
Delete the app.js file from the P.R @sallamTanna

File: client/src/components/Tab/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import Tab from '@material-ui/core/Tab'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles({
+  root: {
+    background: '#5C595B',
+    textTransform: "capitalize",
+    fontSize: "1.1em",
+    color: '#C4C4C4',
+    height: 48,
```
Comment (AbdallahAmmar96) [inline client/src/components/Tab/index.js]: In the height 48 what ?

Comment (sallamTanna) [reply client/src/components/Tab/index.js]: Edited

**Pull request merged**